JSONFormat.go

```diff
@@ -39,7 +45,7 @@ func (j jsonEntry) MarshalJSON() ([]byte, error) {
 		return nil, ErrNoMessage
 	}
 
-	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, j.Message))
+	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, escapeSpecialChars(j.Message)))
```

Comment:
Err.. yes, that's why I wrote it that way. :P The former is dumb simple, the latter is short, but is far more complex.
And to the second question, yes, I would, and I actually do.
And to the third, I actually do build things *that* low level when I like it.

To keep it short, that's how I write code these days. I _do_ skip "mature, reliable" libraries--which way too often are neither. I do write more low-level code. I do read the source code of dependencies, and if they are too generic, too fluffy, too.. magic.. I dump said dependency or rewrite them to my fit. I find lots of "best practices" to be bullshit. I don't like OOP. I don't like SOLID. I don't like pure functional, pure anything. I don't trust anything or anyone.
I got burned too many times, and just don't do that kinda stuff any longer, I can't be bothered.
The only acronym I like lately is "YAGNI".

But, as I said, it ain't my fight. You gotta maintain it, then change it to your liking. What do you care if I wrote it this way or that way?

BTW, Go being low level was a jest. I would argue that C++ is higher level than Go, which in turn is higher level than C, but none of them are low level.